### PR TITLE
perf: Optimize suboptimal code

### PR DESCRIPTION
**Pull request by justraman**
1. 20x faster indexing
2. process metadata in the queue
3. process collection stats in a queue at the end of full sync. 
4. Queue Dashboard
5. fetch account details in cleaner manner
6. fix traits calculation
7. Add Balance Transfer to SNS

### CODE REVIEW

File: .github/workflows/publish_image.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 name: Publish docker image
 
 on:
+  workflow_dispatch:
   push:
+    branches:    
+      - 'perf/optimize-suboptimal-code'
```
Comment (leonardocustodio) [inline .github/workflows/publish_image.yml:7]: I suppose we need to remove this to merge?

**Review by leonardocustodio — CHANGES_REQUESTED** [review]

File: .github/workflows/publish_image.yml (hunk 1 above)
Comment (justraman) [reply .github/workflows/publish_image.yml:7]: I will remove it after we deploy and test it on staging

Comment (leonardocustodio) [reply .github/workflows/publish_image.yml:7]: Why is that? If we are going to merge to master the image should come from master. Unless you are going to deploy before merging.

Comment (justraman) [reply .github/workflows/publish_image.yml:7]: Damian created staging env on AWS, so we will use that for test before merging it to the master.

**Review by leonardocustodio — DISMISSED** [review]

**Review by leonardocustodio — APPROVED** [review]

**Pull request merged**